### DISCUSSION TITLE
End of Life for Reviewpad

### DISCUSSION THREAD

**Discussion by marcelosousa** [cesium/atomic#444] [General]
Hey 👋 

Reviewpad has been acquired by Snyk!  As part of the acquisition, we have joined the Snyk R&D team to help build the next generation of secure code-commit workflows in the Snyk platform.

**As we undertake this new adventure, after January 31st, 2024 (“End of Service Date”) we will not be supporting and maintaining Reviewpad as a standalone product.**

Our team will continue to support your usage of Reviewpad until the End of Service Date to allow time to transition away from the use of Reviewpad. 

We’d like to thank you again for being a supporter of Reviewpad who believed in us and delivered invaluable user feedback that allowed us to build a great application.

For inquiries please email us at [info@reviewpad.com](mailto:info@reviewpad.com).

Best,
**The Reviewpad Team**

**Comment by joaodiaslobo** [comment]
:(
